src/session_builder.c

```diff
@@ -24,6 +26,16 @@ struct session_builder
     signal_context *global_context;
 };
 
+signal_protocol_store_context *session_builder_get_store(session_builder *session_builder)
```

Comment:
Do we really need this function? Other tests call `setup_test_store_context` and get directly `alice_store`

Reply:
removed this function and referenced `alice_store` directly